src/pages/Assets/AssetDetails/AssetHeader/SegwitSelectCard.tsx

```diff
@@ -1,38 +1,46 @@
-import { Button } from '@chakra-ui/react'
-import { ChainTypes } from '@shapeshiftoss/types'
+import { Select } from '@chakra-ui/react'
+import { ChainTypes, UtxoAccountType } from '@shapeshiftoss/types'
+import { useTranslate } from 'react-polyglot'
 import { useDispatch, useSelector } from 'react-redux'
 import { Card } from 'components/Card/Card'
-import { Text } from 'components/Text'
 import { ReduxState } from 'state/reducer'
 import { preferences, supportedAccountTypes } from 'state/slices/preferencesSlice/preferencesSlice'
 
 export const SegwitSelectCard = ({ chain }: { chain: ChainTypes }) => {
   const dispatch = useDispatch()
+  const translate = useTranslate()
   const accountTypes = supportedAccountTypes[chain]
   const currentAccountType = useSelector(
     (state: ReduxState) => state.preferences.accountTypes[chain]
   )
 
   return (
     <Card.Body hidden={!accountTypes}>
-      {accountTypes?.map((accountType, i) => (
-        <Button
-          key={i}
-          size='sm'
-          colorScheme={currentAccountType === accountType ? 'white' : 'blue'}
-          variant='ghost'
-          onClick={() =>
-            dispatch(
-              preferences.actions.setAccountType({
-                key: chain,
-                value: accountType
-              })
-            )
-          }
-        >
-          <Text translation={`assets.assetDetails.assetHeader.${accountType}`} />
-        </Button>
-      ))}
+      <Select
+        size='sm'
+        value={currentAccountType}
+        onChange={option =>
+          dispatch(
+            preferences.actions.setAccountType({
+              key: chain,
+              value: option.target.value
+            })
+          )
+        }
+      >
+        {accountTypes?.map(accountType => (
+          <option
```

Comment:
please used some sort of styled component here that's also ARIA compliant, @reallybeard can provide guidance. the asset selection in swapper might be a good point of reference.

Reply:
@0xdef1cafe I am not sure what you want me to do here. I used the `Select` component from `chakra-ui` which is styled and supposedly respecting ARIA.
Is it about tweaking something in the theme?

Reply:
I think it would prob look better if we used the menu option group for this --- https://chakra-ui.com/docs/overlay/menu#menu-option-groups

Then we can style it in the same way as the other toggles --- variant `ghost-filled`, colorScheme `blue`

Reply:
@0xdef1cafe and @reallybeard  Thank you for your feedbacks and advice!
I implemented the requested changes and indeed it looks a lot better (see screenshots).

![2021-11-23_09-09](https://user-images.githubusercontent.com/5672954/142990170-450cd27e-b96d-4cd0-986e-e779191772b8.png)

![2021-11-23_09-10](https://user-images.githubusercontent.com/5672954/142990187-b8747477-9c95-4b52-a0cd-0799fbe0f21e.png)

![2021-11-23_09-10_1](https://user-images.githubusercontent.com/5672954/142990198-0ca6edc0-46fd-4c25-835f-8255ca195b37.png)